debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresChangeRecordEmitter.java

```diff
@@ -365,7 +365,7 @@ private Table tableFromFromMessage(List<ReplicationMessage.Column> columns, Tabl
         while (itPkCandidates.hasNext()) {
             final String candidateName = itPkCandidates.next();
             if (!combinedTable.hasUniqueValues() && combinedTable.columnWithName(candidateName) == null) {
-                LOGGER.error("Potentional inconsistency in key for message {}", columns);
+                LOGGER.error("Potentional inconsistency in key for message {}", candidateName);
```

Comment:
Let's change this to - 
`LOGGER.error("Potentional inconsistency in key for candidate {}", candidateName);`